dask test failure

it seems `atop` is using an older dask API

```
================================================================================= FAILURES ==================================================================================
____________________________________________________________________________ test_compute[expr0] ____________________________________________________________________________
blaze/compute/tests/test_dask.py:69: in test_compute
    result = compute(expr, dask_ns)
../../../../miniconda/envs/py27/lib/python2.7/site-packages/multipledispatch/dispatcher.py:163: in __call__
    return func(*args, **kwargs)
blaze/compute/core.py:470: in compute
    result = top_then_bottom_then_top_again_etc(expr3, d4, **kwargs)
blaze/compute/core.py:164: in top_then_bottom_then_top_again_etc
    expr2, scope2 = bottom_up_until_type_break(expr, scope, **kwargs)
blaze/compute/core.py:371: in bottom_up_until_type_break
    **kwargs)}
../../../../miniconda/envs/py27/lib/python2.7/site-packages/multipledispatch/dispatcher.py:163: in __call__
    return func(*args, **kwargs)
blaze/compute/dask.py:40: in compute_broadcast
    *concat((dat, tuple(range(ndim(dat))[::-1])) for dat in data))
../../../../code/py/dask/dask/array/core.py:1099: in atop
    numblocks = dict([(a.name, a.numblocks) for a, ind in arginds])
```

--- a/blaze/compute/dask.py
+++ b/blaze/compute/dask.py
@@ -24,7 +24,7 @@ def elemwise_array(expr, *data, **kwargs):
     leaves = expr._inputs
     expr_inds = tuple(range(ndim(expr)))[::-1]
     return atop(curry(compute_it, expr, leaves, **kwargs),
-                next(names), expr_inds,
+                expr_inds,
                 *concat((dat, tuple(range(ndim(dat))[::-1])) for dat in data))
 
 
@@ -36,7 +36,7 @@ try:
         expr_inds = tuple(range(ndim(expr)))[::-1]
         func = get_numba_ufunc(expr)
         return atop(func,
-                    next(names), expr_inds,
+                    expr_inds,
                     *concat((dat, tuple(range(ndim(dat))[::-1])) for dat in data))
 
     def optimize_array(expr, *data):
@@ -64,13 +64,12 @@ def compute_up(expr, data, **kwargs):
                                                  chunk=chunk)
 
     inds = tuple(range(ndim(leaf)))
-    tmp = atop(curry(compute_it, chunk_expr, [chunk], **kwargs),
-               next(names), inds,
-               data, inds)
+    tmp = atop(curry(compute_it, chunk_expr, [chunk], **kwargs), inds, data,
+               inds)
 
     return atop(compose(curry(compute_it, agg_expr, [agg], **kwargs),
                         curry(_concatenate2, axes=expr.axis)),
-                next(names), tuple(i for i in inds if i not in expr.axis),
+                tuple(i for i in inds if i not in expr.axis),
                 tmp, inds)
 
 

